e2e-tests/playwright/lib/src/server/plugin.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Client4} from '@mattermost/client';
+import {PluginManifest} from '@mattermost/types/plugins';
+
+export async function installAndEnablePlugin(
```

Comment:
Items to consider:
- If plugin is already active then no need to force install and activate given same version
- If plugin is already installed, then just enable
- Should cater installation by URL or file

Still more to improve but would be good to port from https://github.com/mattermost/mattermost/blob/master/e2e-tests/cypress/tests/support/api/plugin.js#L57